Resources/Prototypes/Entities/Buildings/computers.yml

```diff
@@ -155,7 +155,7 @@
 - type: entity
   id: ComputerId
   parent: ComputerBase
-  name: ID Card Computer
+  name: id card computer
```

Comment:
`id` is one I'm not sure about. I think lowercase is considered also correct, but ID is preferred. It's just a matter of taste, though.

Reply:
![image](https://user-images.githubusercontent.com/8107459/83130901-51474d80-a0df-11ea-96f3-441341fe7f28.png)

ID seems correct to me.